### DISCUSSION TITLE
Doubt In task 2 - Historical Simulations

### DISCUSSION THREAD

**Discussion by ansenpjames** [cafawo/FinancialDataAnalytics#12] [Q&A] [answered]
Any idea why I am getting 'nan' as output....

dataframe A = logreturns of the ' .GDAXI '
                     B = ewp (log returns of equally weighted assets other than ' .GDAXI ' )

<img width="1217" alt="image" src="https://github.com/cafawo/FinancialDataAnalytics/assets/92854585/a16f985b-55d7-4d5d-88af-70bde9c0a14c">

**Comment by OlgaKuzminichna** [comment, marked as answer]
I believe it is because the very first value is included in return, which is nan
In my code I ask to ignore first line of logreturn
```
log_returns.iloc[1:,0]
```

**Reply by OlgaKuzminichna** [reply]
I thought we get two values, to compare gdaxi with others

**Reply by ansenpjames** [reply]
yeah.. u r right......thanks for the clarification
